pkgs/development/python-modules/paddleocr/default.nix

```diff
@@ -0,0 +1,84 @@
+{ pkgs
+, buildPythonPackage
+, fetchFromGitHub
+, attrdict
+, beautifulsoup4
+, cython
+, fire
+, fonttools
+, lmdb
+, lxml
+, numpy
+, opencv4
+, openpyxl
+, pdf2docx
+, pillow
+, premailer
+, pyclipper
+, pymupdf
+, python-docx
+, rapidfuzz
+, scikit-image
+, shapely
+, tqdm
+, paddlepaddle
+, lanms-neo
+, Polygon3
+}:
+
+let
+
+in buildPythonPackage rec {
+  pname = "paddleocr";
+  version = "2.7.0.0";
+
+  src = fetchFromGitHub {
+    owner = "PaddlePaddle";
+    repo = "PaddleOCR";
+    rev = "df54a7b4223c64a8480380f618fe95de3b8cf73b";
+    sha256 = "H8sBu6+ApquJy/Kr56cYuAuBEz3cDTPh1QutfyzDnG4=";
+  };
+
+  doCheck = false;
+
+  patches = [
+    ./remove-import-imaug.patch
+    ./requirements-txt.patch
+  ];
+
+  propagatedBuildInputs = [
+    attrdict
+    beautifulsoup4
+    cython
+    fire
+    fonttools
+    lmdb
+    lxml
+    numpy
+    opencv4
+    openpyxl
+    pdf2docx
+    pillow
+    premailer
+    pyclipper
+    pymupdf
+    python-docx
+    rapidfuzz
+    scikit-image
+    shapely
+    tqdm
+    paddlepaddle
+    lanms-neo
+    Polygon3
+  ];
+
+  meta = with pkgs.lib; {
+    homepage = "https://github.com/PaddlePaddle/PaddleOCR";
+    license = licenses.asl20;
+    description = "Multilingual OCR toolkits based on PaddlePaddle";
+    longDescription = ''
+      PaddleOCR aims to create multilingual, awesome, leading, and practical OCR
+      tools that help users train better models and apply them into practice.
+    '';
+  };
```

Comment:
There's also the issue of maintainership for all of these packages. My motivation for packaging these tools exists because a project at work uses them, and I've been asked to make it deployable. I only have until the end of this week before I'm off the project, and will likely have no interest in maintaining this outside of that.

Reply:
happy to be added as a maintainer.